retryGet and Loading message

cryogenian opened the pull request:
resolves [SD-1025](https://slamdata.atlassian.net/browse/SD-1025)

@beckyconning @garyb please review

beckyconning commented:
👍

garyb commented:
Looks good, although, the build is failing now? I'll restart it, see if it passes.

cryogenian commented:
Fixed build
* Added special case for `/mount/fs` (it should run only once) 
* Increased timeout in config because some negative tests wait until `Affjax` throws error and now all `GET`s try to take response for 30s.
* Suddenly SD-914 appeared. I worked this around by replacing `sendKeys` with `keys` (which I believe better emulate user behaviour)

jonsterling commented:
@cryogenian Nice job fixing; `keys` is probably a good idea here, nice to know that we have that in our lib. Would be nice to get that added to `purescript-webdriver` some day!

cryogenian commented:
good to merge?

jonsterling commented:
I'm happy with it.

jonsterling commented:
@cryogenian Just one question—Why should `mount/fs` only be run once? (I believe you, I just want to make sure I understand the reason)

cryogenian commented:
It has success when we in database directory. In other cases it returns error. We use it to determine where user is in db or collection or mount.

jonsterling commented:
@cryogenian OK, sounds fine. :+1:
